scss/utilities/_json-api.scss

```diff
@@ -66,44 +71,40 @@
 /// ### Colors
 ///
 /// Each color has an access name and value.
-/// For us, that is often a map-key string,
-/// and Sass hex/rgba output color:
+/// The key will be used to identify the correct data
+/// for a given color-palette,
+/// and the data includes color name:value pairs,
+/// both in string format.
 ///
-/// ```scss
+/// ```
 /// 'brand-colors': (
-///   'brand-orange': '#c75000',
-///   'brand-blue': '#0d7fa5',
+///   'brand-orange': #c75000,
+///   'brand-blue': #0d7fa5,
 /// )
 /// ```
 ///
-/// If you are storing colors in variables,
-/// you may want to use name keys like `$brand-orange`
-/// to represent the proper use of a color.
-///
 /// Color values can be in any valid web-color format --
-/// hex, hsl/a, rgba/a, etc --
-/// converted to a valid JSON string.
+/// hex, hsl/a, rgba/a, etc.
 ///
 /// ### Sizes & Ratios
 ///
 /// Size and ratio data is similar to colors,
 /// organized into top-level groups that may contain
-/// one or more size/ratio key:value pairs:
+/// one or more key:value pairs:
```

Comment:
add a space between `key:` and `value pairs` here, too? Looks especially odd with the `:` after it 
![screen shot 2017-11-27 at 1 16 38 pm](https://user-images.githubusercontent.com/1581694/33284637-5cf45bb2-d375-11e7-9375-4f2b32aabd02.jpg)